application/client/src/app/app.module.ts

```diff
@@ -23,18 +24,24 @@ import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatIconModule } from '@angular/material/icon';
 import { MatListModule } from '@angular/material/list';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+<<<<<<< HEAD
```

Comment:
This file needs to be looked at... there is an unsolved merge conflict going on here